Basic-Car-Maintenance/Shared/Odometer/Views/ReadingView.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ReadingView.swift
+//  Basic-Car-Maintenance
+//
+//  https://github.com/mikaelacaron/Basic-Car-Maintenance
+//  See LICENSE for license information.
+//
+
+import SwiftUI
+
+struct ReadingView: View {
```

Comment:
I'm going to rename this `OdometerRowView`, as opposed to `ReadingView`, to make it more clear

I fixed this